Tests/SwiftFusionTests/Geometry/Pose2Tests.swift

```diff
@@ -181,14 +244,17 @@ final class Pose2Tests: XCTestCase {
     let p5T1 = between(map[4], map[0])
 
     // Test condition: P_5 should be identical to P_1 (close loop)
-    XCTAssertEqual(p5T1.t_.magnitude, 0.0, accuracy: 1e-2)
+    XCTAssertEqual(p5T1.t.magnitude, 0.0, accuracy: 1e-2)
   }
 
   static var allTests = [
     ("testBetweenIdentitiesTrivial", testBetweenIdentitiesTrivial),
     ("testBetweenIdentities", testBetweenIdentities),
     ("testBetweenIdentities", testBetweenIdentitiesRotated),
     ("testBetweenDerivatives", testBetweenDerivatives),
+    ("testDerivativeIdentity", testDerivativeIdentity),
```

Comment:
That’s not automatic?

Reply:
Nope, unfortunately swift can't automatically detect the tests on Linux for some reason. I think I heard someone is working on it, so hopefully this will get improved soon.